### DISCUSSION TITLE
Harbor data migration

### DISCUSSION THREAD

**Discussion by swasthibp** [goharbor/harbor#20749] [Q&A] [answered]
Hi Team,

When migrating harbor from one platform to another platform, can you please let us know what all can be synced via harbor and what cannot be synced? Also, needed information if the transfer of the harbor volume to the new platform can enable a direct migration without any additional configuration.

Can we also know if the replication time is immediate even in case of large images nearly 50-100GBs in size?

An early response would be highly appreciated.

**Comment by zyyw** [comment, marked as answer]
You can create replication rules to sync the images from one Harbor to the other configured in another platform. Other information, like labels, tag retention rule, gc schedule, RBAC, robot account and etc may have to be manually reconfigured.

**Reply by swasthibp** [reply]
what about the projects?

**Reply by zyyw** [reply]
Essentially, there are two types of storage:
* registry 
* database

Images stored in Harbor registry can be sync-ed to another Harbor instance via replication. As for database data, users may need to reconfigure themselves.
